### DISCUSSION TITLE
Modal Overlay background not showing up consistently

### DISCUSSION THREAD

**Discussion by Bond-Addict** [epicmaxco/vuestic-ui#3539] [General]
Not sure if anyone else has experienced this, but occasionally the overlay background color keeps getting dropped. I added a manual class 
```
.va-modal__overlay{
  background-color: rgba(0, 0, 0, 0.6) !important;
}
```

which fixed it for me, but I can't recreate it in a sandbox either. It's rather inconsistent. I've noticed similar situations of component styles not getting applied as well. 

I had to do
```
.va-modal__footer .va-button--normal:nth-of-type(2){
  background-color: $dark-blue !important;
}
``` 
because the Ok button wasn't showing a background, which because it has a white font color made it invisible.

**Comment by m0ksem** [comment]
What `vuestic-ui` version do you use? I've tried to reproduce it, but it looks OK for me in 1.7.0.

**Reply by Bond-Addict** [reply]
^1.6.4

I tried updating to 1.7.0 but all of the select components stopped working so I reverted.
